### ISSUE TITLE
Remaining steps before DC and DC Virtual Lab RC.1 (with fuse)

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#494]
We completed dev testing in https://github.com/phetsims/QA/issues/314 and https://github.com/phetsims/QA/issues/311 and all issues have either been closed, deferred, set to low priority or also exist in the published version.  This issue is to track additional work, if any, before we can start RC.1 testing.

I realized we have not had code review for the new Fuse code.  According to https://github.com/phetsims/circuit-construction-kit-common/issues/317 it looks like @jonathanolson has previously done code reviews for this simulation.  The new code to be reviewed is Fuse.js (125 lines) and FuseNode.js (185 lines).  They are written to match the style of existing components, but I thought I should check if they should be reviewed before RC.1.

Also, sometimes we schedule interviews before RC.  I'm not sure that's warranted or desirable in this case. 

Assigned to @ariel-phet for discussion.

**Comment by samreid** [comment]
In email, @ariel-phet recommended I reach out to @Denz1994 for code review.  I'll create an issue.

**Comment by samreid** [comment]
Code review issue is #395.  @arouinfar do you expect we are doing interviews before RC1?  I recalled we were doing that for Wave Interference but hadn't heard it mentioned for CCK DC/fuse yet--just thought I'd double check.  Close if interviews not necessary.

**Comment by arouinfar** [comment]
@samreid I'm not sure what the timeline is for publishing the fuse. I can certainly add CCK to the Wave Interference interviews, but we're still a few weeks out from that (summer term starts June 3). I was also waiting on Wave Interference interviews until after we share/get feedback from STROBE, in case there are any design changes. 

@kathy-phet @ariel-phet any thoughts on fuse interviews?

**Comment by kathy-phet** [comment]
I'm not sure the fuse by itself needs interviews. I would be comfortable with publishing it without interviews as it seems like a relatively straightforward component addition that is consistent with the other components. But I would pay more attention to this when AC is interviewed on as a check at that point.

**Comment by arouinfar** [comment]
Sounds good @kathy-phet. I think it's pretty safe to table fuse interviews until we get to CCK: AC.

**Comment by arouinfar** [comment]
Seems like this issue can be closed, but back to @samreid to confirm.
